uaa/src/test/java/org/cloudfoundry/identity/uaa/authentication/manager/DynamicZoneAwareAuthenticationManagerTest.java

```diff
@@ -154,7 +154,7 @@ void testNonUAAZoneUaaActiveUaaAuthenticationSuccess() {
     }
 
     @Test
-    void testNonUAAZoneUaaActiveUaaAuthenticationFailure() {
+    void testNonUAAZoneUaaActiveUaaAuthenticationFail() {
```

Comment:
> refactor: rename methods with "Failure" in the name
> * Reserve "FAILURE" for JUnit test fail logs so we won't find these methods in the logs when looking for test failures.

I don't understand this. AFAIK the Junit keyword we see in the test output is `FAILED` not `FAILURE`.

Reply:
Dang it. I'll revert this one.